### PR TITLE
Fixing typos in turtle.rst

### PR DESCRIPTION
**Pull request by Seluj78**
Even more typos. (I'll keep updating the PR as I find more or open new ones)

### CODE REVIEW

File: Doc/library/turtle.rst [hunk 1]
```diff
@@ -1894,7 +1894,7 @@ Input methods
    Pop up a dialog window for input of a number. title is the title of the
    dialog window, prompt is a text mostly describing what numerical information
    to input. default: default value, minval: minimum value for input,
-   maxval: maximum value for input
+   maxval: maximum value for input.
```
Comment (serverwentdown) [inline Doc/library/turtle.rst:1897]: I think this is better rewritten as:

```
... *title* is the title of the dialog window, *prompt* is a text mostly describing what numerical information to input.  *default* is a default value to fill the input with. 

The number input must be in the range  *minval* .. *maxval* (inclusive) if these are given. If outside of this range, a hint is issued and the dialog remains open for correction.
```

**Review by serverwentdown — APPROVED** [review]
LGTM, minor changes.

**Comment by nanjekyejoannah** [comment]
Since there is no activity to add the requested changes, and PR went stale, I will merge as-is for now.

@serverwentdown, you can open a PR for your suggestions if you wish.

**Review by nanjekyejoannah — COMMENTED** [review]
Since there is no activity to add the requested changes, and PR went stale, I will merge as-is for now.

@serverwentdown, you can open a PR for your suggestions if you wish.

**Pull request merged**